slick.grid.js

```diff
@@ -4768,19 +4768,15 @@ if (typeof Slick === "undefined") {
         activeRow = cell.row;
         activeCell = activePosX = activeCell = activePosX = getCellFromNode(activeCellNode);
 
-        $activeCellNode.addClass("active");
-        if (rowsCache[activeRow]) {
-          $(rowsCache[activeRow].rowNode).addClass('active');
-        }
-
```

Comment:
This logic is redundant if `showCellSelection` is `true` and shouldn't be executed if it's `false`

Reply:
I don't agree with removing this, we shouldn't remove a behavior this is already in place.
However can you add an `if` check as I've put on the other line 
```js
if (rowsCache && rowsCache.hasOwnProperty(activeRow) && rowsCache[activeRow].rowNode) {
```

Reply:
But as it stands the codebase has:

`$activeCellNode.addClass("active");
        if (rowsCache[activeRow]) {
          $(rowsCache[activeRow].rowNode).addClass('active');
        }`

followed a few lines down by:

`if (options.showCellSelection) {
        $(activeCellNode).addClass("active");
        $(rowsCache[activeRow].rowNode).addClass("active");
        }`

So this is removing a bug where the `showCellSelection` option is not respected rather than removing desired behavior

Reply:
then I would rather have @6pac commenting on this

Reply:
Hey, decided to merge this as-is.
- the earlier code block adding the active section wasn't in the earlier releases, it appeared after the split columns changes, which leads me to think it's an error as I agree it doesn't really make sense
- while the additional if clause suggested by @ghiscoding  is generally a good idea, it does appear to me that the rowsCache is used as an array (int indexers) so I'm not sure ```hasOwnProperty``` would work as intended, and I think we can safely assume that the ```rowsCache``` object itself is not null

Thanks all for the bugfix and the discussion :-)